README.md

```diff
@@ -426,6 +427,13 @@ If a user agent partitions service workers using this scheme, there is no cross-
 
 [Service workers are disabled](https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Privacy/State_Partitioning) in Firefox when Dynamic Partitioning is enabled, but they are [working on implementing a partitioned service worker solution](https://bugzilla.mozilla.org/show_bug.cgi?id=1495241).
 
+## Security and Considerations
+
+### Security
+
+This proposal offers a small security improvement over third-party/cross-site cookies as currently available on the platform, because they require the `__Host-` prefix and `Secure` attribute.
+This proposal offers a significant privacy improvement over third-party cookies, because it removes a cross-site tracking vector while allowing third-parties to maintain some notion of session/state.
```

Comment:
It's more that this proposal makes it more viable to get the privacy improvement of phasing out third-party cookies in general.

Was there something about how, if some of the limits are picked carelessly, it'd be possible to build a tracking system by correlating when cookies are dropped? That kind of advice for implementations is a good thing to put in a privacy considerations section.

Reply:
> It's more that this proposal makes it more viable to get the privacy improvement of phasing out third-party cookies in general.

Good point, changed.

> Was there something about how, if some of the limits are picked carelessly, it'd be possible to build a tracking system by correlating when cookies are dropped? That kind of advice for implementations is a good thing to put in a privacy considerations section.

Added a short paragraph covering that, thanks for the reminder.